R/list.common.R

```diff
@@ -15,9 +15,13 @@
 #'
 #' foo <- list(x = LETTERS[1:3], y = LETTERS[3:5])
 #' list.common(foo)
-list.common <- function(.data, expr) {
-  if (length(.data) == 0L) return(NULL)
-  expr <- if(missing(expr)) quote(.) else substitute(expr)
+list.common <- function(.data, expr = NULL) {
+
+  if (!length(.data)) {
+    return(NULL)
+  }
+
+  expr <- ifelse(is.null(expr), quote(.), substitute(expr))
```

Comment:
This line seems to break the [example](https://travis-ci.org/renkun-ken/rlist/jobs/83249943#L2512). `is.null(expr)` forces the evaluation of the promise `expr = .` so that `.` is not substituted for non-standard evaluation. 

`list.common(x, .)` is a usage intending to get the common elements of a list or vector by mapping each element to itself (i.e. identity), just like the usage of `list.map(x, .)`, a degenerate case to map each element of `x` to itself.